audit/projects/k8s-conform/buckets/k8s-conform-provider-openstack/bucketpolicyonly.txt

```diff
@@ -0,0 +1,4 @@
+Bucket Policy Only setting for gs://k8s-conform-provider-openstack:
```

Comment:
I see no trace of this in the codebase? Why does it exist?

Reply:
looks like we have a bunch of these. i only run scripts, i don't know enough to meddle in the UI :) I believe i was re-running some of the conform buckets 
```
[dims@dims-a01 07:15] ~/go/src/k8s.io/k8s.io ⟩ rg -i "Bucket Policy Only" | wc -l
     244
```

In this instance i think i was trying to re-run scripts again to see how to help with:
https://github.com/kubernetes/test-infra/issues/20914

Reply:
https://github.com/kubernetes/k8s.io/pull/1676/files/5552daee41483e00b09c7ad5b1c57d699ab9848d..176f6e37790b71ce30ef9b57ddf78804abdfe1b3#r585516339

Reply:
Ultimately I would like for us to have this enabled across the org, and enforced via an org policy

per-object ACLs are much trickier to audit and enforce

Reply:
Sorry, my point in tagging @dims was that I can't find any trace of these projects in git.  Did someone forget to send a PR?

Reply:
i know we cleaned some stuff up in:
https://github.com/kubernetes/k8s.io/issues/1311#issuecomment-778507615

Only reference to `k8s-conform-provider-openstack` i can find is:
https://github.com/theopenlab/openlab/issues/691

may be @chrigl knows more?

Reply:
@thockin it is in the codebase
- these buckets (and the k8s-conform project) are created by https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/ensure-conformance-storage.sh
- the PR adding this bucket was https://github.com/kubernetes/k8s.io/pull/1645
- dims ran the script afterwards, like he was supposed to

Reply:
I apologize.  I got bitten by master/main - I had not resynced this copy in a while and was trying to sync master and not noticing that it failed.

Indeed, it is in the tree.  Mea culpa, my apologies.